templates/email/default/problem-confirm-not-sending.html

```diff
@@ -0,0 +1,33 @@
+[%
+
+email_summary = "You need to confirm your " _ site_name _ " report before it can be made public.";
+email_columns = 2;
+problem_map_url = inline_image(report.static_map, "map.png");
+
+PROCESS '_email_settings.html';
+
+INCLUDE '_email_top.html';
+
+%]
+
+<th style="[% td_style %][% primary_column_style %]" id="primary_column">
+  <h1 style="[% h1_style %]">Please confirm your&nbsp;report</h1>
+  <p style="[% p_style %]">Please click on the link below to confirm that you want your report to appear
+on [% site_name %], despite not being sent to the
+council.</p>
+  <p style="[% p_style %]">Your report will also appear on the [% site_name %] website.</p>
+  <p style="margin: 20px auto; text-align: center">
+    <a style="[% button_style %]" href="[% token_url %]">Yes, send my report</a>
```

Comment:
This button should say 'Publish my report' or similar, as it's not being sent anywhere.